build.py

```diff
@@ -197,12 +197,17 @@ def set_config(filename, config, new_value):
     with open(filename) as fp:
         lines = fp.readlines()
 
+    found = False
     with open(filename, 'w') as fp:
         for line in lines:
             if line.startswith(config + '='):
+                found = True
                 line = '%s=%s\n' % (config, new_value)
             fp.write(line)
 
+    assert found, '"%s" is not found in %s' % (config, filename)
```

Comment:
Should we use `assert`, or should we use something that survives if Python optimization is turned on? (This isn't pytest with its weird manipulation of `assert`s...)

(I think it's OK this way, just noting it here.)